doc/contributor_experience_team.rst

```diff
@@ -15,7 +15,7 @@
     </div>
     <div>
     <a href='https://github.com/jmloyola'><img src='https://avatars.githubusercontent.com/u/2133361?v=4' class='avatar' /></a> <br />
-    <p>Juan Martín Loyola</p>
+    <p>Juan Martin Loyola</p>
```

Comment:
On his webpage, it is written with the diacritic.

Reply:
This is the name from is github profile. https://github.com/jmloyola. He must have changed that recently.
(This page is automatically generated)